web/src/components/SearchFeedback.tsx

```diff
@@ -13,6 +15,22 @@ const Container = styled.div`
   align-items: center;
 `
 
+const CenteredContainer = styled.div`
+  text-align: center;
+`
+
+const SemiBoldText = styled.p`
+  font-weight: 600;
+`
+
+const MiddleText = styled.p`
+  padding-bottom: 1rem;
```

Comment:
:question: Hmm, did I miss something or why did you switch to rem? Is this already part of the new design system? Otherwise I'd prefer to keep it consistent.

Reply:
You're right, not sure why I did that 😅